frontend/server/src/Controllers/Course.php

```diff
@@ -3026,9 +3026,14 @@ public static function getStudentsInformationForSmarty(
         [
             'allProgress' => $allProgress,
             'problemTitles' => $problemTitles,
-        ] = \OmegaUp\DAO\Courses::getStudentsInCourseWithProgressPerAssignment(
-            $course->course_id,
-            $course->group_id
+        ] = \OmegaUp\Cache::getFromCacheOrSet(
+            \OmegaUp\Cache::SCHOOL_STUDENTS_PROGRESS,
+            "{$courseAlias}",
```

Comment:
```suggestion
            $courseAlias,
```

aquí no hay necesidad de cadenificar esto. igual en L3087.